R/RLearner_regr_randomForest.R

```diff
@@ -104,7 +119,7 @@ bootstrapStandardError = function(.learner, .model, .newdata, ...) {
     pred.boot = matrix(pred.boot, nrow = nrow(.newdata), ncol = R, byrow = TRUE)
   }
   var.boot = apply(pred.boot, 1, var) - bias
-  var.boot[var.boot <= 0] = 0
+  var.boot = pmax(var.boot, 0)
```

Comment:
This changes `var.boot` -- before only the values < 0 were changed, now all but one of them are. Was this a bug in the previous version?